mission.md

```diff
@@ -0,0 +1,17 @@
+# Mission
+
+We are on a mission to change the way that analytics is done. We believe that [analysts should work more like software engineers](https://medium.com/fishtown-analytics/building-a-mature-analytics-workflow-the-analyst-collective-viewpoint-7653473ef05b), and we spend our time thinking about, implementing, building tools for, and training analysts on this workflow. This mission animates everything about Fishtown Analytics: the way we work, the work we choose to do, the clients we work with, and the people we hire.
+
+We care about this mission for a very simple reason: we want to help analysts be more effective. We’re analysts ourselves, and have personally witnessed a dramatic increase in our productivity by implementing this workflow. We want to help others do the same.
```

Comment:
This is a super-interesting point, and it's one that I thought about as well. Here's where I came out; I'm interested to hear if you buy this.

I think this document should not be an exposition on anything and everything potentially related to our mission. That could be rather a lot. So, I think it's important to rely on shortcuts. When I referenced our mission, I linked the canonical blog post that explains the point of view. I think it's possible that other areas of this page could also be great targets for further exposition, and I think that this is a good one.

I personally don't feel that our reasons for "why we care" are all that interesting and differentiated. For me, personally, it really does come down to a) scratching my own itch, and b) identifying large a problem to solve where I can actually add something. I do think that there are positive benefits to individuals' careers, organizations' success, and you could even make some grander "world-changing" argument, in that data-driven decisions are more globally maximalist. Those arguments may have merit but they are not, I don't believe, why we got into doing this nor are they why we still go to work in the morning.

It's possible that the "why" inevitably gets personal, which is not to say that it doesn't matter, only that we will inevitably be a collection of people who bring their own "whys". This feels, to me, to be appropriate.

Sorry, I just said a lot. This wasn't at all to shoot you down--rather, it was because I think this is a great point and I thought a lot about it.